cloud/pkg/synccontroller/objectsync.go

```diff
@@ -90,8 +86,9 @@ func sendEvents(err error, nodeName string, sync *v1alpha1.ObjectSync, resourceT
 	if CompareResourceVersion(objectResourceVersion, sync.Status.ObjectResourceVersion) > 0 {
 		// trigger the update event
 		klog.V(4).Infof("The resourceVersion: %s of %s in K8s is greater than in edgenode: %s, send the update event", objectResourceVersion, resourceType, sync.Status.ObjectResourceVersion)
-		msg := buildEdgeControllerMessage(nodeName, sync.Namespace, resourceType, sync.Spec.ObjectName, model.UpdateOperation, obj)
-		beehiveContext.Send(commonconst.DefaultContextSendModuleName, *msg)
+		if msg := buildEdgeControllerMessage(nodeName, sync.Namespace, resourceType, sync.Spec.ObjectName, model.UpdateOperation, obj); msg != nil {
+			beehiveContext.Send(commonconst.DefaultContextSendModuleName, *msg)
+		}
```

Comment:
Does the modification in this file related to the issue this pr fixed? If not, we can move it to another PR :)

Reply:
i found related PR had been merged: https://github.com/kubeedge/kubeedge/pull/3570